src/http.js

```diff
@@ -170,4 +171,13 @@ export default class HTTP {
     }
     this.events.emit("backoff", backoffMs);
   }
+
+  _checkForRetryAfterHeader(status, headers) {
+    let retryAfter = headers.get("Retry-After");
+    if (!retryAfter) {
+      return;
+    }
+    retryAfter = parseInt(retryAfter, 10);
+    this.events.emit("retry-after", retryAfter);
```

Comment:
We can convert this delay into date by using the current timestamp and adding the delay.